src/components/TextField/__tests__/TextField.spec.js

```diff
@@ -1,9 +1,13 @@
 import SbTextField from '..'
-import { mount } from '@vue/test-utils'
+import { mount, createLocalVue } from '@vue/test-utils'
+import VueMask from 'v-mask'
+const localVue = createLocalVue()
+localVue.use(VueMask)
```

Comment:
You can use the setup logic from jest to make the `VueMask` a global configuration: you can start reading [here](https://jestjs.io/docs/configuration#setupfilesafterenv-array). You need to create a file to use in this configuration and, in this file, you can import the Vue and add the VueMask plugin as a global plugin as we do in the main.js file.

Reply:
Uuunmmm.... Thanks @emanuelgsouza, I will read here!